fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/functions/scalar/CurrentCatalog.java

```diff
@@ -0,0 +1,54 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.nereids.trees.expressions.functions.scalar;
+
+import org.apache.doris.catalog.FunctionSignature;
+import org.apache.doris.nereids.trees.expressions.functions.AlwaysNotNullable;
+import org.apache.doris.nereids.trees.expressions.functions.ExplicitlyCastableSignature;
+import org.apache.doris.nereids.trees.expressions.shape.LeafExpression;
+import org.apache.doris.nereids.trees.expressions.visitor.ExpressionVisitor;
+import org.apache.doris.nereids.types.VarcharType;
+
+import com.google.common.collect.ImmutableList;
+
+import java.util.List;
+
+/**
+ * ScalarFunction 'CurrentCatalog'.
+ */
+public class CurrentCatalog extends ScalarFunction
```

Comment:
u need to do constant fold in fe, like the database() function do
```
org.apache.doris.nereids.rules.expression.rewrite.rules.FoldConstantRuleOnFE#visitDatabase
```

Reply:
> u need to do constant fold in fe, like the database() function do
> 
> ```
> org.apache.doris.nereids.rules.expression.rewrite.rules.FoldConstantRuleOnFE#visitDatabase
> ```

ok, let me fix code